src/mumble/Settings.h

```diff
@@ -186,6 +188,15 @@ struct OverlaySettings {
 	friend bool operator!=(const OverlaySettings &lhs, const OverlaySettings &rhs);
 };
 
+struct Profiles {
+	static const std::string s_default_profile_name;
+	static const int s_current_settings_version;
```

Comment:
Why do we need this as a static member inside `Profiles` instead of existing only as a constant in the `JSONSerialization.cpp` implementation? It shouldn't be needed anywhere else :eyes:

Reply:
I want to be able to handle the default profile differently in the UI (frontend). E.g. disable the ability to delete or rename it. Also: Sort it before anything else in the drop down menu.

Reply:
I was talking about the settings format version :)